airflow/providers/databricks/operators/databricks.py

```diff
@@ -1056,13 +1073,24 @@ def monitor_databricks_job(self) -> None:
             )
         if not run_state.is_successful:
             raise AirflowException(
-                "Task failed. Final state %s. Reason: %s",
-                run_state.result_state,
-                run_state.state_message,
+                f"Task failed. Final state {run_state.result_state}. Reason: {run_state.state_message}"
```

Comment:
@pankajkoti , I updated this exception call. The reason is that AirflowException doesn't actually just extend the Base Exception class. Hence, all the extra arguments we send are not actually used to parse the final exception message.

example:

```
>>> from airflow.exceptions import AirflowException
>>> raise AirflowException("This is %s", "Hello")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
airflow.exceptions.AirflowException: ('This is %s', 'Hello')
```

Reply:
Thanks for fixing this @rawwar